fluent-bundle/src/types/mod.rs

```diff
@@ -181,6 +181,19 @@ impl<'source> FluentValue<'source> {
             FluentValue::None => "".into(),
         }
     }
+
+    pub fn into_owned<'a>(&self) -> FluentValue<'a> {
+        match self {
+            FluentValue::String(str) => FluentValue::String(Cow::from(str.to_string())),
+            FluentValue::Number(s) => FluentValue::Number(s.clone()),
+            FluentValue::Custom(s) => {
+                let new_value: Box<dyn FluentType + Send> = s.duplicate();
```

Comment:
Question: Does the compiler want you to put this on the stack as `new_value` and provide a type or could you turn this into a one-line? It's unclear from just reading the code if there is a reason for the extra verbosity.

`FluentValue::Custom(s.duplicate())`